test/Dialect/FIRRTL/parse-basic.fir

```diff
@@ -1525,6 +1525,10 @@ circuit Layers:
     ; CHECK-NEXT:   }
     ; CHECK-NEXT: }
 
+  ; CHECK: firrtl.layer @WithDirectory1 bind attributes {output_file = #hw.output_file<"foo/">}
+  layer WithDirectory1, bind, "foo":
+  ; CHECK: firrtl.layer @WithDirectory2 bind attributes {output_file = #hw.output_file<"foo/">}
+  layer WithDirectory2, bind, "foo/":
```

Comment:
It just occurred to me that the directory is really only there for the "bind" convention. This may be cleaner as something like the following:

```
layer WithDirectory1, bind<"foo">:
layer WithDirectory1, bind("foo"):
layer WithDirectory1, bind{"foo"}:
```

This is related to this _not_ making sense for the inline convention.

Reply:
I see what you're saying, I can do this if you'd like.

My plan was to "not parse" or "not accept" the output directory after parsing an inline convention.